build.sbt

```diff
@@ -60,49 +27,12 @@ Global / onLoad := (Global / onLoad).value.andThen { s =>
   s
 }
 
-// this overrides interplay release settings with some adaptations for working with sbt-dynver
-// moreover, it contain only bits necessary for twirl, nothing else
-lazy val releaseSettings: Seq[Setting[_]] = Seq(
-  // Release settings
-  releaseCrossBuild := false,
-  releaseProcess := {
-    import ReleaseTransformations._
-
-    def ifDefinedAndTrue(key: SettingKey[Boolean], step: State => State): State => State = { state =>
-      Project.extract(state).getOpt(key in ThisBuild) match {
-        case Some(true) => step(state)
-        case _          => state
-      }
-    }
-
-    Seq[ReleaseStep](
-      checkSnapshotDependencies,
-      runClean,
-      releaseStepCommandAndRemaining("+test"),
-      releaseStepTask(playBuildExtraTests in thisProjectRef.value),
-      releaseStepCommandAndRemaining("+publishSigned"),
-      releaseStepTask(playBuildExtraPublish in thisProjectRef.value),
-      ifDefinedAndTrue(playBuildPromoteBintray, releaseStepTask(bintrayRelease in thisProjectRef.value)),
-      ifDefinedAndTrue(playBuildPromoteSonatype, releaseStepCommand("sonatypeBundleRelease")),
-      pushChanges
-    )
-  }
-)
```

Comment:
I think this was lost on the refactor.
Should this be on `project/Publish.scala`?

Reply:
We only publish to bintray (without promoting), we have nothing to push to git. So basically all this block can go.

Reply:
Even the `+test`?

Ah, ofc! This will be a job in travis where tests have already run!